### PR TITLE
further changes

### PR DESCRIPTION
**Pull request by anushamogili**
https://github.com/junkerm/specmate/commit/d9d9eb25996418dc5ef06bb79800835ef31f226a

### CODE REVIEW

**Comment by anushamogili** [comment]
Refactor the get to refer Requirement and model

File: web/src/app/modules/views/side/modules/links-actions/components/links-actions.component.ts [hunk 1]
```diff
@@ -16,8 +16,8 @@ export class LinksActions {
 
     public isCollapsed = false;
 
-    public _requirement: Requirement;
-    public _model: IContainer;
+    //public _requirement: Requirement;
+    //public _model: IContainer;
```
Comment (munterkalmsteiner) [inline web/src/app/modules/views/side/modules/links-actions/components/links-actions.component.ts]: How are these changes related to the implementation of feature 7?

File: web/src/app/modules/views/main/authentication/modules/login/components/login.component.html [hunk 2]
```diff
@@ -60,6 +60,21 @@ <h1 class="card-title text-center mb-4 mt-1">{{'login' | translate}}</h1>
             </button>
             <span id="login-language-dropdown" class="pull-right"><language-chooser></language-chooser></span>
         </div>
+        <div class="form-group">
+              <!----<img  height="200" *ngFor="let url of urls" [src]="url">-->
+<input type='file' (change)="onSelectFile($event)">
```
Comment (munterkalmsteiner) [inline web/src/app/modules/views/main/authentication/modules/login/components/login.component.html]: Why is here an input field? For variant 1 of this feature, the user shall not be able to define the background image.

File: web/src/app/modules/views/main/authentication/modules/login/components/login.component.html [hunk 3]
```diff
@@ -60,6 +60,21 @@ <h1 class="card-title text-center mb-4 mt-1">{{'login' | translate}}</h1>
...
+            <fieldset style="padding: 5px;border: black solid 1px;">
+                    <legend style=" padding-left: 5px; font-size: 15px;font-weight: bolder; border: none; width: 180px;">Images Uploaded By you</legend>
+                    <span  *ngFor="let url of urls"style="border">
+                        <img  [src]="url" (click)="setBGImage(url)" *ngIf="url!=null || url!=undefined"  style="height:100px;width:100px;border: black solid 1px;" >
+                    </span>
+                    <img [src]="bgImage" style="height:100px;width:100px">
+                </fieldset>
+</div>
+    </div>
+</div>
```
Comment (munterkalmsteiner) [inline web/src/app/modules/views/main/authentication/modules/login/components/login.component.html]: Please indent the code correctly.

**Comment by munterkalmsteiner** [comment]
Closing this PR as the feature was already merged from another team. Thanks!
